main.go

```diff
@@ -85,26 +86,29 @@ func main() {
 
 	// add flags related to this operator
 	var (
-		metricsAddr             string
-		probeAddr               string
-		pprofAddr               string
-		agentImage              string
-		autoInstrumentationJava string
-		autoAnnotationConfigStr string
-		webhookPort             int
-		tlsOpt                  tlsConfig
+		metricsAddr               string
+		probeAddr                 string
+		pprofAddr                 string
+		agentImage                string
+		autoInstrumentationJava   string
+		autoInstrumentationPython string
+    autoAnnotationConfigStr string
+		webhookPort               int
```

Comment:
nit: Is this just a weird formatting issue? Did you run `make fmt`?

Reply:
this is what i get for resolving merge conflicts from the github ui sigh. I knew I should've used cli